src/com/magento/idea/magento2plugin/init/ConfigurationManager.java

```diff
@@ -0,0 +1,247 @@
+import com.intellij.util.PlatformUtils;
+import com.jetbrains.php.config.PhpProjectConfigurable;
+import com.jetbrains.php.config.library.PhpIncludePathManager;
+import com.jetbrains.php.ui.PhpUiUtil;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.function.Function;
+import com.magento.idea.magento2plugin.project.Settings;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class ConfigurationManager {
+    private static final ConfigurationManager INSTANCE = new ConfigurationManager();
+
+    public static ConfigurationManager getInstance() {
+        return INSTANCE;
+    }
+
+    private ConfigurationManager() {
+    }
+
+    public void refreshIncludePaths(Settings.State newState, Project project) {
+        if (!project.isDefault() && newState.isPluginEnabled() && !newState.isDoNotAskContentConfigAgain()) {
+            VirtualFile magentoFile = getMagentoFile(newState);
+            if (magentoFile == null) {
+                return;
+            }
+
+            if (VfsUtilCore.isAncestor(project.getBaseDir(), magentoFile, false)) {
+                return;
+            }
+
+            boolean isModuleInsideMagento = VfsUtilCore.isAncestor(magentoFile, project.getBaseDir(), false);
+            if (isModuleInsideMagento) {
+                Module[] modules = ModuleManager.getInstance(project).getModules();
+                if (modules.length == 1) {
+                    Module module = modules[0];
+                    boolean isMagentoIncluded = isFileInsideModule(magentoFile, module);
+                    if (!isMagentoIncluded) {
+                        suggestToChangeContentRoots(magentoFile, project, module);
+                    }
+                }
+            } else {
+                boolean isMagentoIncluded = isInIncludePath(magentoFile, project);
+                if (!isMagentoIncluded) {
+                    suggestToAddToIncludePath(magentoFile, project);
+                }
+            }
+        }
+
+    }
+
+    private static void suggestToChangeContentRoots(@NotNull VirtualFile magentoFile, @NotNull Project project, @NotNull Module module) {
+        String message = "For Magento 2 containing plugins inside it's better to add whole Magento 2 to project.";
+        Function<Notification, AnAction> fixAction = (notification) -> {
+            return new DumbAwareAction("Fix") {
+                public void actionPerformed(@NotNull AnActionEvent e) {
+                    notification.expire();
+                    ModuleRootModificationUtil.updateModel(module, (model) -> {
+                        ContentEntry[] contentEntries = model.getContentEntries();
+                        ContentEntry rootEntry = null;
+                        ContentEntry[] entries = contentEntries;
+                        int length = contentEntries.length;
+
+                        for(int i = 0; i < length; ++i) {
```

Comment:
```suggestion
                        for (int i = 0; i < length; ++i) {
```

Reply:
@eduard13 
Thank you for the review. Could you please recheck?